pandas/core/indexes/base.py

```diff
@@ -5290,6 +5291,29 @@ def _add_logical_methods_disabled(cls):
 Index._add_comparison_methods()
 
 
+class IndexGroupbyGroups(dict):
+    def __repr__(self):
+        from pandas.compat import u
```

Comment:
FYI the earliest we would implement this is 0.25 at which point we'll drop Py2, so no need to use compat code here

Reply:
Thanks, removed it.